### PR TITLE
Cleanup capacity related fields from facility

### PR DESCRIPTION
**Pull request by bodhish**
## Proposed Changes

Lets remove facility capacity and beds. 

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

## Summary by CodeRabbit

- **New Features**
	- Streamlined facility creation process by removing unnecessary steps and fields.
	- Enhanced user interface for facility management by retaining essential components.

- **Bug Fixes**
	- Improved error handling and validation in facility creation forms.
	- Removed components related to bed and staff capacity management, simplifying the user experience.

- **Chores**
	- Cleaned up codebase by removing obsolete components and associated logic.
	- Removed several API routes related to hospital bed management and triage functionalities.
	- Updated type declarations to reflect changes in data structures.
	- Simplified test suites by removing tests related to doctor and bed capacity management.
	- Updated Cypress commands and utilities for improved test interactions.

### CODE REVIEW

**Comment by bodhish** [comment]
@nihal467 can you cleap up the spec ☝️

**Comment by nihal467** [comment]
@bodhish @rithviknishad 
image

can you remove all the related exports as well from the facility page

File: cypress/pageobject/utils/constants.ts [hunk 1]
```diff
@@ -1,10 +1,122 @@
...
   window.crypto.getRandomValues(array);
   const randomNum = (array[0] % 900000000) + 100000000;
   return "9" + randomNum.toString();
 }
 
-export function generateEmergencyPhoneNumber(): string {
+function generateEmergencyPhoneNumber(): string {
   return generatePhoneNumber();
 }
+
```
[the remark this answers is not in the document]
Comment (nihal467) [reply cypress/pageobject/utils/constants.ts:11]: there is no failure in generation

File: src/components/Facility/FacilityList.tsx [hunk 2]
```diff
@@ -157,30 +157,17 @@ export const FacilityList = () => {
...
+            <ExportMenu
+              exportItems={[
+                {
+                  label: "Facilities",
+                  route: routes.downloadFacility,
+                  filePrefix: "facilities",
+                },
+              ]}
+            />
+          </div>
```
[the remark this answers is not in the document]
Comment (nihal467) [reply src/components/Facility/FacilityList.tsx:170]: resolved it check it

File: cypress/pageobject/Facility/FacilityHome.ts [hunk 3]
```diff
@@ -10,7 +9,7 @@ class FacilityHome {
   }
 
   navigateToFacilityHomepage() {
-    cy.visit("/facility");
+    cy.awaitUrl("/facility");
```
[the remark this answers is not in the document]
Comment (nihal467) [reply cypress/pageobject/Facility/FacilityHome.ts:12]: it's a command custom function, it is having cy.visit in it

**Review by nihal467 — APPROVED** [review]

**Review by rithviknishad — APPROVED** [review]

**Pull request merged**